File: java-poc/src/main/java/io/craigmiller160/querydslpoc/java/data/repository/EmployeeRepository.java
package io.craigmiller160.querydslpoc.java.data.repository;

import io.craigmiller160.querydslpoc.java.data.entity.Employee;
import java.util.UUID;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface EmployeeRepository extends JpaRepository<Employee, UUID> {}

File: kotlin-poc/src/main/kotlin/io/craigmiller160/querydslpoc/kotlin/data/repository/EmployeeRepository.kt
package io.craigmiller160.querydslpoc.kotlin.data.repository

import io.craigmiller160.querydslpoc.kotlin.data.entity.Employee
import java.util.UUID
import org.springframework.data.jpa.repository.JpaRepository

interface EmployeeRepository : JpaRepository<Employee, UUID>
